pyproject.toml

```diff
@@ -1,10 +1,23 @@
-#[project]
-#requires-python = ">=3.9"
-
 [build-system]
-requires = ["setuptools>=62.4.0"]
+requires = [
+    "Babel>=2.10.0",
+    "PyInstaller==6.12.0",
+    "pytest>=8.1",
+    "setuptools>=62.4.0",
```

Comment:
In my test I had to use at least `setuptools>=setuptools>=66.1.0` for this to properly process.

Also not sure about the rest of the dependencies. They are all not strictly needed for a `pyhon -m build`. We only need them for a more complete build. Shall we keep them out of this PR and add them as needed later?